image/image.go

```diff
@@ -456,6 +470,13 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 		if info.Base != "" && !local.hasName(snaps, info.Base) {
 			return fmt.Errorf("cannot add snap %q without also adding its base %q explicitly", name, info.Base)
 		}
+		// warn about missing default providers
+		dps := neededDefaultProviders(info)
+		for _, dp := range dps {
+			if !local.hasName(snaps, dp) {
+				fmt.Fprintf(Stderr, "WARNING: the %q default content provider %q is not getting installed.", info.InstanceName(), dp)
```

Comment:
Would it be useful to have an option to treat warnings as errors in this scenario?

Reply:
Maybe/probably - but also YAGNI, happy to add it once we get a feature request for it.